apps/crn-server/src/handlers/research-output/algolia-index-research-output-handler.ts

```diff
@@ -33,6 +33,9 @@ export const indexResearchOutputHandler =
 
         const data = {
           ...researchOutput,
+          source: researchOutput.workingGroups?.length
```

Comment:
This should belong in the modal if it isn't there already. We don't want to duplicate this logic in lots of places incase it changes